src/custom/components/swap/EthFlow/containers/EthFlowModal/hooks/useSetupEthFlow.ts

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState } from 'react'
+import { ApprovalState } from 'hooks/useApproveCallback'
+import { useSetAtom } from 'jotai'
+import {
+  ethFlowContextAtom,
+  resetEthFlowContextAtom,
+  updateEthFlowContextAtom,
+} from '../../../state/ethFlowContextAtom'
+import { ActivityDescriptors, ActivityStatus } from 'hooks/useRecentActivity'
+import { useAtomValue } from 'jotai/utils'
+import { EthFlowActions } from './useEthFlowActions'
+import { delay } from 'utils/misc'
+import { EthFlowState } from '../../../typings'
```

Comment:
Not only this file so to avoid spam I'll write only here.
Also, this is a nitpick and definitely not a release blocker.

Use absolute/aliased paths rather than relative, at least for anything that's not in the same folder